pages/pil/exemptions/content/index.js

```diff
@@ -13,5 +13,11 @@ module.exports = {
     exempt: {
       required: 'You need to select yes or no.'
     }
+  },
+  action: {
+    repeat: {
+      add: 'Add another',
+      remove: 'Remove'
+    }
```

Comment:
can we bring this up a level so it can be shared by both exemptions and training?

Reply:
done